src/server/oasisapi/analyses/tasks.py

```diff
@@ -175,6 +174,31 @@ def store_file(reference, content_type, creator, required=True, filename=None):
             raise e
 
 
+def get_worker_model(name, supplier, version):
+    from src.server.oasisapi.analysis_models.models import AnalysisModel
```

Comment:
in general it is not ideal to import in the function, because import will be called several time and it slightly slow things down.
Just pointing at it, maybe there is a good reason for this